ipaserver/install/server/replicainstall.py

```diff
@@ -1030,7 +1031,13 @@ def promote_check(installer):
         except ipaclient.ntpconf.NTPConfigurationError:
             pass
 
-    api.bootstrap(in_server=True, context='installer')
+    env = Env()
+    env._bootstrap(context='installer', log=None)
+    env._finalize_core(**dict(constants.DEFAULT_CONFIG))
+
+    # pylint: disable=no-member
```

Comment:
NACK: you disabled pylint globally until end of file, enable it later or put it to the right line